src/cmd/create.go

```diff
@@ -699,9 +699,14 @@ func pullImage(image, release, authFile string) (bool, error) {
 	}
 
 	if promptForDownload {
+		infoSize, err := skopeo.Inspect(imageFull)
```

Comment:
I see that you are now passing `imageFull`, which doesn't have the `docker://` prefix.  That's better.  However, I don't see the prefix being added inside `skopeo.Inspect`.

Reply:
Now I am passing the string ```docker://``` inside the ```skopeo.Inspect()``` method, I hope that works better.

Reply:
No, it should be inside the body of the `skopeo.Inspect` function.

Like I said [before](https://github.com/containers/toolbox/pull/1208#discussion_r1073452041), the `docker://` prefix is something that `skopeo inspect` needs.  The caller of `skopeo.Inspect` doesn't need it and doesn't need to know about it.  So, it should be inside `skopeo.Inspect`.